helm/object-storage-operator/templates/rbac.yaml

```diff
@@ -1,82 +1,72 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ include "resource.default.name"  . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+  name: {{ include "resource.default.name" . }}
 rules:
   - apiGroups:
-      - ""
+      - infrastructure.cluster.x-k8s.io
     resources:
-      - pods
+      - awsclusters
+      - awsclusterroleidentities
     verbs:
-      - "*"
-  - apiGroups:
-      - ""
-    resources:
-      - configmaps
-      - namespaces
-    verbs:
-      - create
-      - update
-      - delete
       - get
       - list
+      - watch
   - apiGroups:
-      - ""
+      - objectstorage.giantswarm.io
     resources:
-      - serviceaccounts
+      - buckets
+      - buckets/status
     verbs:
-      - create
+      - watch
+      - get
+      - list
+      - update
+      - patch
   - apiGroups:
-      - "networking.k8s.io"
+      - coordination.k8s.io
     resources:
-      - networkpolicies
+      - leases
     verbs:
+      - get
       - create
+      - update
   - apiGroups:
-      - "rbac.authorization.k8s.io"
+      - ""
     resources:
-      - clusterrolebindings
+      - events
     verbs:
       - create
-  - apiGroups:
-      - apiextensions.k8s.io
-    resources:
-      - customresourcedefinitions
-    verbs:
-      - "*"
-  - nonResourceURLs:
-      - "/"
-      - "/healthz"
-    verbs:
-      - get
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ include "resource.default.name"  . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+  name: {{ include "resource.default.name" . }}
 subjects:
   - kind: ServiceAccount
-    name: {{ include "resource.default.name"  . }}
-    namespace: {{ include "resource.default.namespace"  . }}
+    name: {{ include "resource.default.name" . }}
+    namespace: {{ include "resource.default.namespace" . }}
 roleRef:
   kind: ClusterRole
-  name: {{ include "resource.default.name"  . }}
+  name: {{ include "resource.default.name" . }}
   apiGroup: rbac.authorization.k8s.io
 ---
+{{- if not .Values.global.podSecurityStandards.enforced }}
 {{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```

Comment:
if we are using `Values.global.podSecurityStandards.enforced`, do we need 
```
{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
```
?

Reply:
Not specifically but this does not hurt imo